be/src/exec/vectorized/file_scanner.cpp

```diff
@@ -287,9 +305,22 @@ Status FileScanner::create_random_access_file(const TBrokerRangeDesc& range_desc
             break;
         } else {
             BrokerFileSystem fs_broker(address, params.properties);
-            ASSIGN_OR_RETURN(auto broker_file, fs_broker.new_random_access_file(range_desc.path));
-            src_file = std::shared_ptr<RandomAccessFile>(std::move(broker_file));
-            break;
+            if (config::use_local_filecache_for_broker_random_access_file) {
```

Comment:
I suggest judging which scenarios are suitable for this optimization. The scenarios we know include scenarios with small files and many columns. Others, such as large files, are obviously not applicable.

Reply:
It's hard to define size of small files and count of many colums. How about put this configuration in broker load opt_properties, and user can determine whether to use it?

Reply:
it's ok